test/pseudo-tty/no_dropped_stdio.js

```diff
@@ -12,4 +12,13 @@ let out = ('o'.repeat(48) + '\n').repeat(20);
 out += 'o'.repeat(24) + 'O';
 
 process.stdout.write(out);
-process.exit(0);
+
+// In AIX, the child exits even before the python parent
+// can setup the readloop. Provide a reasonable delay.
+if (common.isAix) {
+  setTimeout(function() {
+    process.exit(0);
+  }, 200);
```

Comment:
@addalex - the background for this change is[here](https://github.com/nodejs/node/issues/9728#issuecomment-280304651)
There exists a bug in python in AIX, but the current issue and its mitigation is outside of that bug.
I am not sure I understand why this would break the purpose of the test - can you please explain? 
The sequence of child-parent interaction is thus:
1. Parent turns child's std streams into pseudo terminals, and spawns the child.
2. child writes data into its streams and exits. [ data of differing length, and sequence, in different tests]
3. Parent sets up a read loop and continuously reads, until it gets EIO (on child termination)
4. The read data is then compared with the expected data, with the help of a companion file.

The issue is: there is a race condition between child's exit and parent's read logic - if child exits too early, the parent will end up loosing all the data - in theory this is possible in all platforms, but AIX seem to be the only victim, and hence this workaround.

It is not a pre-req that the child starts only when the parent is ready. If the child stays alive, so does its fds, and hence parent can continue to read from it.

Please let me know if I am missing anything.